include/open62541/client_subscriptions.h

```diff
@@ -102,6 +102,10 @@ UA_Client_Subscriptions_delete_async(UA_Client *client,
 UA_StatusCode UA_EXPORT UA_THREADSAFE
 UA_Client_Subscriptions_deleteSingle(UA_Client *client, UA_UInt32 subscriptionId);
 
+/* Retrieve the user supplied subscription contexts */
+UA_StatusCode UA_EXPORT UA_THREADSAFE
+UA_Client_Subscriptions_getUserContexts(UA_Client *client, UA_UInt32 subscriptionId, void **subContext, UA_UInt32 monitorId, void **monitorContext);
```

Comment:
Split this into separate methods for Subscription and MonitoredItem.
Also provide both _get and _set methods.

Also call it _getContext instead of _getUserContext.
Then this is consistent with the existing API. Such as UA_Server_setAdminSessionContext.
https://github.com/open62541/open62541/blob/31947a93608e6e7d7132f06573ad00c263641b7b/include/open62541/server.h#L1115